tests/sm_int/sm_int.c

```diff
@@ -1001,12 +1036,17 @@ TEST_FUNCTION(STATE_and_API)
                 }
             }
 
+            LogInfo("went to state =%" PRI_MU_ENUM "; and called =%" PRI_MU_ENUM " cleaner thread might already have run", MU_ENUM_VALUE(SM_STATES, (SM_STATES)(i + SM_CREATED)), MU_ENUM_VALUE(SM_APIS, (SM_APIS)(j + SM_OPEN_BEGIN)));
```

Comment:
>cleaner [](start = 83, length = 7)

poetry: or maybe the switchesToCreated is the cleaner thread?

Reply:
Absolutely corrected.

---
In reply to: [432052699](https://github.com/Azure/azure-c-util/pull/45#discussion_r432052699) [](ancestors = 432052699)